src/main/java/org/csanchez/jenkins/plugins/kubernetes/GarbageCollection.java

```diff
@@ -41,6 +44,9 @@ public class GarbageCollection extends AbstractDescribableImpl<GarbageCollection
     private transient Set<String> namespaceSet;
     private int timeout;
 
+    private static Long RECURRENCE_PERIOD = SystemProperties.getLong(
+            GarbageCollection.class.getName() + ".recurrencePeriod", TimeUnit.MINUTES.toSeconds(1));
```

Comment:
Would be nice to have `SystemProperties.getDuration` that would parse `250ms`, `5s`, `10m`, etc.

Reply:
Yes, too bad `Duration.parse` takes `ISO-8601` format which is not really human readable.